static/js/tools/map/chart_options.tsx

```diff
@@ -30,6 +30,8 @@ import {
 } from "./util";
 import { DataPointMetadata } from "./chart_loader";
 
+const NO_PER_CAPITA_TYPES = ["medianValue"];
```

Comment:
most things with "FractionOf" or "Rate" are good candidates to add here too.

Reply:
I don't think FractionOf or Rate are values for the stat type predicate though .. I checked a few statistical variables that are rates or fractionOf, but their statType is just "measuredValue"

Reply:
gotcha - right, those are baked into the sv dcids.